lgc/include/lgc/state/AbiUnlinked.h

```diff
@@ -63,6 +63,16 @@ const static char DescriptorOffset[] = "doff_";
 //  * 1: descriptor pointer should be loaded from the spill table
 const static char DescriptorUseSpillTable[] = "dusespill_";
 
+// Whether the descriptor pointer comes from a spill table.
+// The format is: "dusespill_X_Y" where:
+// * X is the descriptor set number
+// * Y is the binding number
+//
```

Comment:
This seems to be duplicated from above the previous string.

Reply:
Right, I forgot to fix that up.